active_directory_ldap/komand_active_directory_ldap/connection/connection.py

```diff
@@ -20,7 +20,10 @@ def connect(self, params):
         port = params.get(Input.PORT)
         user_name = params.get(Input.USERNAME_PASSWORD).get('username')
         password = params.get(Input.USERNAME_PASSWORD).get('password')
-        self.logger.info(f'Connecting to {host}{port}')
+        if host.find(':') != -1:
+            host = host.split(':')
+            host = host[0]
```

Comment:
Would be good log here to something like `Port was provided in hostname, using value from Port field instead`